Can't find _stdin/_stdout in M1 macOS

```
$ ./run-examples.sh                                                austral master
+ dune build
+ compile examples/ffi FFI 'aHello, world!\n'
+ ./austral compile examples/ffi/FFI.aui,examples/ffi/FFI.aum --entrypoint=Example.FFI:main --output=testbin
Compiler call tree printed to calltree.html
Error:
  Title: Command Line Arguments Error
  Module:
    [unknown]
  Location:
    [no span available]
  Description:
    Failed to run C compiler.

    Command: `cc /var/folders/x8/8s64cxbx0yj16d3shdxmm5gr0000gn/T/austral_2d2a94.c -fwrapv -lm -o testbin`

    Exit code: 1

    Standard output:


    Standard error:
    /var/folders/x8/8s64cxbx0yj16d3shdxmm5gr0000gn/T/austral_2d2a94.c:84:14: warning: declaration of built-in function 'fprintf' requires inclusion of the header <stdio.h> [-Wbuiltin-requires-header]
      extern int fprintf(void* stream, const char* format, ...);
                 ^
    /var/folders/x8/8s64cxbx0yj16d3shdxmm5gr0000gn/T/austral_2d2a94.c:98:14: warning: declaration of built-in function 'fprintf' requires inclusion of the header <stdio.h> [-Wbuiltin-requires-header]
      extern int fprintf(void* stream, const char* format, ...);
                 ^
    /var/folders/x8/8s64cxbx0yj16d3shdxmm5gr0000gn/T/austral_2d2a94.c:152:16: warning: incompatible redeclaration of library function 'memmove' [-Wincompatible-library-redeclaration]
      extern void* memmove(void* destination, void* source, size_t count);
                   ^
    /var/folders/x8/8s64cxbx0yj16d3shdxmm5gr0000gn/T/austral_2d2a94.c:152:16: note: 'memmove' is a builtin with type 'void *(void *, const void *, unsigned long)'
    /var/folders/x8/8s64cxbx0yj16d3shdxmm5gr0000gn/T/austral_2d2a94.c:158:16: warning: incompatible redeclaration of library function 'memcpy' [-Wincompatible-library-redeclaration]
      extern void* memcpy(void* destination, void* source, size_t count);
                   ^
    /var/folders/x8/8s64cxbx0yj16d3shdxmm5gr0000gn/T/austral_2d2a94.c:158:16: note: 'memcpy' is a builtin with type 'void *(void *, const void *, unsigned long)'
    /var/folders/x8/8s64cxbx0yj16d3shdxmm5gr0000gn/T/austral_2d2a94.c:211:1: warning: non-void function does not return a value in all control paths [-Wreturn-type]
    }
    ^
    /var/folders/x8/8s64cxbx0yj16d3shdxmm5gr0000gn/T/austral_2d2a94.c:3284:10: warning: expression result unused [-Wunused-value]
            ((au_unit_t)(au_printf("true")));
             ^          ~~~~~~~~~~~~~~~~~~~
    /var/folders/x8/8s64cxbx0yj16d3shdxmm5gr0000gn/T/austral_2d2a94.c:3286:10: warning: expression result unused [-Wunused-value]
            ((au_unit_t)(au_printf("false")));
             ^          ~~~~~~~~~~~~~~~~~~~~
    /var/folders/x8/8s64cxbx0yj16d3shdxmm5gr0000gn/T/austral_2d2a94.c:3318:18: warning: expression result unused [-Wunused-value]
                    ((au_unit_t)(au_printf("%c", *((au_nat8_t*) au_array_index(array_tmp_ref0, i, sizeof(au_nat8_t))))));
                     ^          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    8 warnings generated.
    Undefined symbols for architecture arm64:
      "_stdin", referenced from:
          _au_stdin in austral_2d2a94-f442d1.o
         (maybe you meant: _au_stdin)
      "_stdout", referenced from:
          _au_stdout in austral_2d2a94-f442d1.o
         (maybe you meant: _au_stdout)
    ld: symbol(s) not found for architecture arm64
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
  Code:
    [no span available]
```

--- a/lib/prelude.c
+++ b/lib/prelude.c
@@ -54,8 +54,13 @@ au_span_t au_make_span_from_string(const char* data, size_t size) {
 }
 
 void* au_stdout() {
+#if defined(__APPLE__)
+  extern void* __stdoutp;
+  return __stdoutp;
+#else
   extern void* stdout;
   return stdout;
+#endif
 }
 
 void* au_stderr() {
@@ -75,8 +80,13 @@ void* au_stderr() {
 }
 
 extern void* au_stdin() {
+#if defined(__APPLE__)
+  extern void* __stdinp;
+  return __stdinp;
+#else
   extern void* stdin;
   return stdin;
+#endif
 }
 
 au_unit_t au_abort_internal(const char* message) {

